src/main/java/io/cdap/plugin/batch/source/ftp/FTPBatchSource.java

```diff
@@ -170,6 +170,18 @@ public static class FTPBatchSourceConfig extends PluginConfig implements FileSou
             + "read the data.")
     private String schema;
 
+    @Macro
+    @Nullable
+    @Description("The delimiter to use if the format is 'delimited'. The delimiter will be ignored if the format "
+      + "is anything other than 'delimited'.")
+    private String delimiter;
+
+    @Macro
+    @Nullable
+    @Description("Whether to treat content between quotes as a value. This value will only be used if the format " +
+      "is 'csv', 'tsv' or 'delimited'. The default value is false.")
+    protected Boolean enableQuotedValues;
```

Comment:
where are these params used?

Reply:
they're used by the format plugins, but they won't show up in the source UI unless we also put them in the source config

Reply:
https://github.com/cdapio/hydrator-plugins/blob/a7653a1400f849e888ae01784c3fca47be8d5975/format-delimited/src/main/java/io/cdap/plugin/format/delimited/input/DelimitedConfig.java#L66